src/Logic/DarkTheme.cs

```diff
@@ -56,19 +57,30 @@ private static List<T> GetSubControls<T>(Control c)
             return menus.Cast<T>().ToList();
         }
 
-        public static void SetDarkTheme(Control form, int iterations = 5)
+        private static bool _isConfigUpdated;
+
+        public static void SetDarkTheme(Control ctrl, int iterations = 5)
         {
-            Configuration.Settings.General.SubtitleBackgroundColor = BackColor;
-            Configuration.Settings.General.SubtitleFontColor = ForeColor;
-            Configuration.Settings.VideoControls.WaveformBackgroundColor = BackColor;
-            Configuration.Settings.VideoControls.WaveformGridColor = Color.FromArgb(62, 62, 60);
+            if (!_isConfigUpdated)
```

Comment:
The **SetDarkTheme** will be called recursively, so add guard to avoid redundant re-assignment.